JS/Lesson16/gulcin/script.js

```diff
@@ -197,16 +361,14 @@ function checkGrade(grade) {
 		case 'F':
 			console.log(uppercaseGrade + ' Failing');
 			break;
-		default:
-			console.log('Input value is not correct');
-			break;
 	}
+	
 }
 checkGrade("a");
 checkGrade("D");
 checkGrade("Z");
-checkGrade("A--");
-checkGrade("B+");
+checkGrade("A-");
+checkGrade("B+");  // couldnt find extra solution for this other than writing extra cases explicitly :( 
```

Comment:
Grades could be something like: "A", "B--", "D+" etc. The common thing between all of them is that the first char is always the grade letter that you need. So in order to only get the letter you can do something like this: `const uppercaseGrade = grade[0].toUpperCase();`

Reply:
Now I got the point, We just switch the first char, and the rest of the string doesn't affect the condition.